docs/snapshotting/versioning.md

```diff
@@ -3,13 +3,27 @@
 This document describes how Firecracker persists its state across multiple versions, diving deep into the snapshot format, encoding, compatibility and limitations. 
 
 ## Introduction
-The design behind the snapshot implementation enables version tolerant save and restore across multiple Firecracker versions which we call a version space. For example, one can pause a microVM, save it to disk with Firecracker version **0.26.0** and later load it in Firecracker version **0.24.0**. It also works  in reverse: Firecracker version **0.26.0** loads what  **0.24.0** saves. 
+The design behind the snapshot implementation enables version tolerant save and restore across multiple Firecracker versions which we call a version space. For example, one can pause a microVM, save it to disk with Firecracker version **0.23.0** and later load it in Firecracker version **0.24.0**. It also works  in reverse: Firecracker version **0.23.0** loads what  **0.24.0** saves. 
 
-Below is an example graph showing backward and forward snapshot compatibility for a version space formally represented as `[v2; vn]`. This is the general picture, but keep in mind that future corner cases might remove edges from this graph.
+Below is an example graph showing backward and forward snapshot compatibility. This is the general picture, but keep in mind that when adding new features some version translations would not be possible.
 
 ![Version graph](
 ../images/version_graph.png?raw=true
 "Version graph")
+A non-exhaustive list of how cross-version snapshot support can be used:
+
+Example scenario #1 - load snapshot from older version: 
+* Start Firecracker v0.23 → Boot microVM → Pause → CreateSnapshot(snap) → kill microVM
+* Start Firecracker v0.24 → LoadSnapshot → Resume → *Workload continues*
+
+Example scenario #2 - save snapshot new Firecracker and load in older version: 
+* Start Firecracker v0.24 → Boot microVM → Workload starts → Pause → CreateSnapshot(snap, “0.23”) → kill microVM
+* Start Firecracker v0.23 → LoadSnapshot(snap) → Resume → Workload continues
+
+Example scenario #3 - save snapshot new Firecracker and load in older version:
```

Comment:
```suggestion
Example scenario #3 - circular version dependency:
```

Reply:
there is no circular dependency. I've upated to "load snapshot in older version"